client/src/main/java/envoy/client/ui/controller/ChatScene.java

```diff
@@ -167,7 +177,24 @@ private void initialize() {
 		clip.setArcHeight(43);
 		clip.setArcWidth(43);
 		clientProfilePic.setClip(clip);
-
+		
+		Platform.runLater(() -> {
+			if(client.isOnline()) {
+				try {
+					contactSearchTab.setContent(FXMLLoader.load(new File("src/main/resources/fxml/ContactSearchTab.fxml").toURI().toURL()));
+					groupCreationTab.setContent(FXMLLoader.load(new File("src/main/resources/fxml/GroupCreationTab.fxml").toURI().toURL()));
+				} catch (Exception e2) {
```

Comment:
As this is an exception that can only occur in development, we might consider wrapping it as a `RuntimeException` instead.

```suggestion
				} catch (IOException e2) {
```